dotnet/src/Functions/Functions.OpenAPI/OpenAI/KernelOpenAIPluginExtensions.cs

```diff
@@ -132,7 +132,11 @@ public static class KernelOpenAIPluginExtensions
         try
         {
             pluginJson = JsonNode.Parse(openAIManifest)!;
-            openAIAuthConfig = pluginJson["auth"].Deserialize<OpenAIAuthenticationConfig>()!;
+            JsonSerializerSettings settings = new()
```

Comment:
Please add a comment describing the reason the Newtonsoft library used instead of System.Text.Json one.